nanoFramework.Json/JsonObject.cs

```diff
@@ -16,79 +16,11 @@ internal class JsonObject : JsonToken
     {
         private readonly Hashtable _members = new();
 
-        public JsonProperty this[string name]
```

Comment:
Isn't this useful to access properties in code? Like this:
`payloadObject["niceproperty"]`

I don't want to to get rid of useful methods just because they are not covered in Unit Test and, therefore, _seem_ to be unused.

Adding a couple of Unit Tests would be in order, though.

Reply:
It's inside internal class without any references, so it's not used in JSON project. 
In my opinion - if it's internal and not used it should be deleted (to minimalize deployment size of dll).

Reply:
What about exposing it? Is it not useful to have this?
I mean the main goal is to serialize & deserialize, no doubts here. Just wondering if this could be useful for quick & dirty access to the properties. When one is doing "bind" navigation on the payload. Or when the payload schema is not known.

Or maybe this is already accessible through the hastable and friends... 🤔

Reply:
JSONObject is internal and it is never returned from any public method. It's used only in 
```csharp
  private static JsonObject ParseObject(ref LexToken token)
        {
            var result = new JsonObject();
```
So there is no way to expose indexer method without making ParseObject method public.